.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,17 +1,18 @@
 ## 🗒️ Description
 <!-- Brief description of the changes introduced by this PR -->
 
-## 🔗 Related Issues
-<!-- Reference any related issues using the GitHub issue number (e.g., Fixes #123) -->
+## 🔗 Related Issues or PRs
+<!-- Reference any related issues using the GitHub issue number (e.g., Fixes #123). Default is N/A. -->
+N/A.
 
 ## ✅ Checklist
+<!-- Please check off all required items. For those that don't apply remove them accordingly. -->
 
 - [ ] All: Set appropriate labels for the changes.
+- [ ] All: PR title adheres to the [repo standard](https://eest.ethereum.org/main/getting_started/contributing/?h=contri#commit-messages-issue-and-pr-titles) - it will be used as the squash commit message and should start `type(scope): `.
 - [ ] All: Considered squashing commits to improve commit history.
```

Comment:
This is only relevant to merging PRs and only a maintainer can merge. And, perhaps most relevant, our merge policy now enforces squash and merge as a merge policy.
```suggestion
```